ddp-workspace/projects/ddp-mpc/src/app/components/data-release/data-release.component.ts

```diff
@@ -0,0 +1,245 @@
+                <div class="PageLayout">
+                    <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                        <h1 class="PageHeader-title" translate>
+                            DataRelease.Title
+                        </h1>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <article class="PageContent">
+            <div class="PageLayout">
+                <div class="row NoMargin">
+                    <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                        <section class="PageContent-section">
+                            <div class="row NoMargin Left">
+                                <button mat-button color="primary"
+                                        class="ButtonFilled Button--rect"
+                                        (click)="openDisclaimerDialog()"
+                                        [innerHTML]="'DataRelease.ViewDataButton' | translate">
+                                </button>
+                            </div>
+                            <div class="row topMarginMedium">
+                                <p class="PageContent-text" translate>
+                                    DataRelease.MainText.Section1
+                                </p>
+                                <p class="PageContent-text" translate>
+                                    DataRelease.MainText.Section2
+                                </p>
+                                <p class="PageContent-text" translate>
+                                    DataRelease.MainText.Section3
+                                </p>
+                                <h2 class="PageContent-subtitle" translate>DataRelease.InformationList.Title</h2>
+                                <ul class="PageContent-ul">
+                                    <li class="PageContent-text" translate>DataRelease.InformationList.Item1</li>
+                                    <li class="PageContent-text" translate>DataRelease.InformationList.Item2</li>
+                                    <li class="PageContent-text" translate>DataRelease.InformationList.Item3</li>
+                                    <li class="PageContent-text" translate>DataRelease.InformationList.Item4</li>
+                                </ul>
+                                <p class="PageContent-text">
+                                    <span translate>DataRelease.InformationList.Text</span>
+                                    <a [href]="dataEmailHref" class="Link">{{ dataEmail }}</a>.
+                                </p>
+                                <h2 class="PageContent-title" translate>DataRelease.BioPortal.Title</h2>
+                                <p class="PageContent-text">
+                                    <a class="Link" (click)="openDisclaimerDialog()" translate>DataRelease.BioPortal.Link</a>
+                                </p>
```

Comment:
The dialog opened by this link text appears in gray:
![Screen Shot 2020-03-02 at 11 04 26 AM](https://user-images.githubusercontent.com/29984380/75694221-2bdcac00-5c76-11ea-81f6-152282dec4a3.png)

Reply:
Also, I've noticed that OK button should be on the right side.